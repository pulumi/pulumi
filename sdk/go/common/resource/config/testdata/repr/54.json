{"my:name":["a","b","c","value"]}