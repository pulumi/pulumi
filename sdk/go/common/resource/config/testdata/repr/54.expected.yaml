decrypt:
  my:name: '["a","b","c","value"]'
paths:
  my:name:
    value:
    - a
    - b
    - c
    - value
    string: '["a","b","c","value"]'
    redacted: '["a","b","c","value"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAOf+ENgAEBnN0cmluZwwDAAFhBnN0cmluZwwDAAFiBnN0cmluZwwDAAFjBnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: true
  my:name[0]:
    value: a
    string: a
    redacted: a
    object: DhAABnN0cmluZwwDAAFh
    secure: false
    isObject: false
  my:name[1]:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:name[2]:
    value: c
    string: c
    redacted: c
    object: DhAABnN0cmluZwwDAAFj
    secure: false
    isObject: false
  my:name[3]:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
